stepprogress/src/main/java/com/funrisestudio/stepprogress/StepProgressView.kt

```diff
@@ -505,6 +505,63 @@ class StepProgressView : ViewGroup, View.OnClickListener {
         return stepProgress.isAllDone()
     }
 
+    /**
+     * Change title text padding
+     */
+    fun setTextTitlePadding(size: Float) {
+        textTitlePadding = size
+        requestLayout()
+    }
+
+    /**
+     * Change node title text size
+     */
+    fun setNodeTitleSize(size: Int) {
+        textNodeTitleSize = size
+        resetView()
+        invalidate()
+    }
+
+    /**
+     * Change text node  size
+     */
+    fun setNodeTextSize(size: Int) {
+        textNodeSize = size
+        requestLayout()
+    }
+
+    /**
+     * Change node height
+     */
+    fun setNodeHeight(size: Float) {
+        nodeHeight = size
+        requestLayout()
+    }
+
+    /**
+     * Change node text color
+     */
+    fun setNodeTextColor(@ColorInt color: Int) {
+        textNodeColor = color
+        children.forEach {
```

Comment:
Call resetView() instead of updating color manually. That will ensure that you don't change color for nodes that are not active

Reply:
fixed